scripts/sync_tx_translations.js

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env babel-node
+
+/**
+ * @fileoverview
+ * Script to pull translations from transifex and generate the editor-msgs file.
+ * Expects that the project and resource have already been defined in Transifex, and that
+ * the person running the script has the the TX_TOKEN environment variable set to an api
+ * token that has developer access.
+ */
+
+const args = process.argv.slice(2);
+
+const usage = `
+ Pull supported language translations from Transifex. Usage:
```

Comment:
Yeah, it's so much clearer to me what this is doing because you use "Pull" here.